docs/cow-protocol/concepts/flashloans/integrators.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 2
+---
+
+# How to specify an order with flashloan
+
+The flashloan is encoded into the [appData](/cow-protocol/reference/core/intents/app-data)'s metadata as an optional object. This information serves as a hint for the solver, but the solver ultimately can modify this data in order to make the operation more optimal.
```

Comment:
> serves as a hint for the solver, but the solver ultimately can modify this data

The solver does not modify the data inside the appdata. It could only decide to ignore it if it finds a better way to do things.